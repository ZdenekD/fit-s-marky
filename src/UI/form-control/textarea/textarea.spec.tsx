import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../../__test__/utils/helpers';
import Textarea from '.';

describe('Textarea', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Textarea name="textarea" label="Textarea" />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-textarea');

        expect(component.exists()).toBe(true);
    });
});
