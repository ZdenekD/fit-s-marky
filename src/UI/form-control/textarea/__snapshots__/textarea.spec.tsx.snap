// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea match snapshot 1`] = `
<div class="control ">
  <label for="_5innv1"
         class="label "
  >
    Textarea
  </label>
  <div class="wrapper">
    <textarea id="_5innv1"
              name="textarea"
              class="textarea   "
              data-test="component-textarea"
    >
    </textarea>
  </div>
</div>
`;
